memif-attach/main.go

```diff
@@ -83,23 +109,22 @@ func createMemif(ctx context.Context, conn api.Connection, socketID uint32, mode
 		log.Entry(ctx).Fatalln("ERROR: memifCreate failed:", err)
 	}
 	log.Entry(ctx).Infof("Memif interface created")
-	log.Entry(ctx).Infof("swIfIndex"+
-		"Role"+
-		"SocketID",
+	log.Entry(ctx).Infof("swIfIndex %v\n"+
+		"Role %v\n"+
+		"SocketID %v\n",
```

Comment:
INFO[0000] Memif interface created
INFO[0000] swIfIndex 0
Role MEMIF_ROLE_API_SLAVE
SocketID 2

Reply:
Change 
```go
	if isClient {
		role = memif.MEMIF_ROLE_API_SLAVE
	}
```

to 
```go
	if !isClient {
		role = memif.MEMIF_ROLE_API_SLAVE
	}
```